src/sql/workbench/services/notebook/common/interface.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+
+export interface INotebookInput {
```

Comment:
INotebookInput is also defined here https://github.com/microsoft/azuredatastudio/blob/780ca84f9a5104591fbae4231c9323f14f6aab28/src/sql/workbench/services/notebook/browser/notebookService.ts#L162

Reply:
+1, I'd suggest removing it from the other place and just having this one here